src/dex-ui/pages/dao/ProposalDetailsPage/ProposalDetailsHeader.tsx

```diff
@@ -17,6 +17,9 @@ export function ProposalDetailsHeader(props: ProposalDetailsHeaderProps) {
   if (daoType === DAOType.GovernanceToken) {
     to = `/daos/${Paths.DAOs.GovernanceToken}/${daoAccountId}/${Paths.DAOs.Proposals}`;
   }
+  if (daoType === DAOType.NFT) {
```

Comment:
we can use `ternary` operator here to avoid using any `let` variable.

Reply:
Done.